Sources/SwiftInspectorVisitors/GenericRequirementVisitor.swift

```diff
@@ -31,14 +31,20 @@ public final class GenericRequirementVisitor: SyntaxVisitor {
 
   public override func visit(_ node: SameTypeRequirementSyntax) -> SyntaxVisitorContinueKind {
     genericRequirements.append(GenericRequirement(node: node))
-    return .visitChildren
+    // We only care about siblings, not children.
+    return .skipChildren
   }
 
   public override func visit(_ node: ConformanceRequirementSyntax) -> SyntaxVisitorContinueKind {
     genericRequirements.append(GenericRequirement(node: node))
-    return .visitChildren
+    // We only care about siblings, not children.
```

Comment:
Let's uncatify this one too.

Reply:
1caf37d99c1cbd555b1f4e26b08de4eaf18fc800